docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -265,6 +265,8 @@ tieredstore:
     low: 0.7
 ```
 
+> There 2 supported volume `type`: `hostPath` and `emptyDir`. `hostPath` mounts the directory from the host node’s filesystem into your Pod. The benifits are that data can be kept even the worker restarted. `emptyDir` is first created when a Pod is assigned to a Node, and exists as long as that Pod is running on that node. As the name says, it is initially empty. The benifit of empty dir is that it can set memory limit in Kubernetes level.
```

Comment:
"benifits" -> "benefits", "benifit -> "benefit"

Reply:
@cheyang I suggest keeping this explanation as short as possible and leave a link to [hostPath](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath) and emptyDir K8s doc. The more we explain here the more we possibly have to update in the future.

For example, let's not explain what emptyDir does.